homeworks/oleg.borodko_olegborodko/homework_5/task_4.js

```diff
@@ -0,0 +1,24 @@
+const people = [
+    { id: 1, name: 'Nick', friends: [2, 5, 6] },
+    { id: 2, name: 'John', friends: [1, 3] },
+    { id: 3, name: 'Mike', friends: [2, 5] },
+    { id: 4, name: 'Janny', friends: null },
+    { id: 5, name: 'Edward', friends: [1, 3] },
+    { id: 6, name: 'Jeen', friends: [5, 1] },
+];
+
+const getPeople = function (userId) {
+    if (userId <= 0 || userId > 6) {
+        return null;
+    }
+
+    const { friends } = people[userId - 1].friends;
+
+    if (friends === null) {
+        return [];
+    }
+
+    return people[userId - 1].friends.map(numeric => people[numeric - 1]);
+};
+
```

Comment:
avoid any math with `id`s as numbers, `id` is just an identifier, in real life it could be anything like `iddghj45hj35b`

wrong approach